Library/Homebrew/extend/os/mac/extend/ENV/super.rb

```diff
@@ -5,13 +5,19 @@ module Superenv
   extend T::Sig
 
   class << self
+    # The location of Homebrew's shims on macOS.
+    # @public
+    def shims_path
+      HOMEBREW_SHIMS_PATH/"mac/super"
+    end
+
     undef bin
 
     # @private
     def bin
       return unless DevelopmentTools.installed?
 
-      (HOMEBREW_SHIMS_PATH/"mac/super").realpath
+      shims_path.realpath
     end
```

Comment:
This should probably be in the generic case now, unless the extra `DevelopmentTools.installed?` check is somehow important.

Reply:
> This should probably be in the generic case now, unless the extra `DevelopmentTools.installed?` check that's only there on macOS is somehow important.

I don't know that it isn't so I'd rather leave it as-is for now 👍🏻